console/console-init/ui/src/components/FormAndJson/FormAndJson.tsx

```diff
@@ -64,10 +64,7 @@ const FormAndJson: React.FunctionComponent<IFormAndJsonProps> = ({
           />
         </>
       ) : (
-        <>
-          <ToggleOffIcon color="black" size={IconSize.lg} onClick={onToggle} />
-          {console.log(detailInJson)}
-        </>
+        <>{console.log(detailInJson)}</>
```

Comment:
why do we have console.log here. remove if not necessary.

Reply:
This is a poc component.